pages/osx/nettop.md

```diff
@@ -0,0 +1,35 @@
+# nettop
+
+> Display dynamic real-time information about the network.
+
+- Start nettop monitoring TCP and UDP sockets from all interfaces:
+
+`nettop`
+
+- Start nettop monitoring TCP sockets from Loopback interfaces:
+
+`nettop -m tcp -t loopback`
+
+- Start nettop monitoring specific process:
+
+`nettop -p {{process_id|process_name}}`
+
+- Start nettop displaying summary only for per-process:
+
+`nettop -P`
```

Comment:
Who thought it might be a good idea to use both `-p` and `-P` as arguments? 😆

Reply:
Without these two options, this command will show the detail for all processes, a long page will be displayed on screen.
With `-P` option, user can roughly monitor all processes status, while with `-p` option, user can check all connection detail belong to one process, so I think these two options might to be use separately 😄

Reply:
No, I was talking about that they are very easy to confuse, since they look very similar. 😄

Reply:
Oh, I misunderstood before. Yes, they are really easy to confuse without man page 😅